fastapi_sqla/base.py

```diff
@@ -30,7 +30,6 @@ def setup(app: FastAPI):
                 functools.partial(sqla.add_session_to_request, key=key)
             )
         else:
-            assert has_asyncio_support, asyncio_support_err
```

Comment:
Removed because of B017, and because we can never actually encounter this situation since engine creation will fail if asyncpg isn't present.

Reply:
So it still fails on app boot and not on the first request right?

Reply:
Yes